--- gammu-1.29.90/CMakeLists.txt~	2011-02-10 15:30:41.000000000 +0100
+++ gammu-1.29.90/CMakeLists.txt	2011-02-14 10:39:42.502867523 +0100
@@ -471,7 +471,7 @@
     CMAKE_COMPILER_IS_CYGWIN OR CMAKE_COMPILER_IS_GNUCXX OR CLANG_COMPILER)
     # Check for extra compiler flags we want to use
     if (NOT GAMMU_VERSION_PATCH LESS 90)
-        MACRO_TUNE_COMPILER("-Werror")
+        #MACRO_TUNE_COMPILER("-Werror")
     endif (NOT GAMMU_VERSION_PATCH LESS 90)
 
     #
