--- gammu-1.02.0/common/device/serial/ser_unx.c.orig	2005-05-03 12:32:34.000000000 +0000
+++ gammu-1.02.0/common/device/serial/ser_unx.c	2005-10-29 14:41:14.291744934 +0000
@@ -249,6 +249,7 @@
 		case 1152000:	speed2 = B1152000;	break;
 		case 1500000:	speed2 = B1500000;	break;
 		case 2000000:	speed2 = B2000000;	break;
+#ifdef B2500000
 		case 2500000:	speed2 = B2500000;	break;
 		case 3000000:	speed2 = B3000000;	break;
 		case 3500000:	speed2 = B3500000;	break;
@@ -256,6 +257,7 @@
 #endif
 #endif
 #endif
+#endif
 	}
 
     	/* This should work on all systems because it is done according to POSIX */
