--- cfg/Makefile.cfg~  2005-07-27 15:00:05.000000000 +0200
+++ cfg/Makefile.cfg       2005-07-28 11:52:09.000000000 +0200
@@ -10,7 +10,7 @@
 ECHO             = /bin/echo
 CC               = gcc
 CFLAGS           = -O2 -fPIC -Wall -Wno-format-y2k -pthread
-LDLIBS           =   -pthread
+LDLIBS           =   -lmysqlclient -lz -lcrypt -pthread -lbluetooth
 INSTALL          = /usr/bin/install
 FIND             = /bin/find
 RPMBUILD         = /bin/rpm
